cli/foundation-models/system/finetune/image-classification/multiclass-classification/hftransformers-fridgeobjects-multiclass-classification.sh

```diff
@@ -150,9 +111,10 @@ fi
 # # If you want to use a HuggingFace model, specify the inputs.model_name instead of inputs.mlflow_model_path.path like below
 # inputs.model_name=$huggingface_model_name
 
-huggingface_parent_job=$( az ml job create \
+huggingface_parent_job_name=$( az ml job create \
   --file "./hftransformers-fridgeobjects-multiclass-classification-pipeline.yaml" \
   $workspace_info \
+  --query name -o tsv \
```

Comment:
name or display_name? In jquery, we are looking for display_name.

Reply:
Thanks Gaurav. Name is working on ci and on windows, both.